src/app/pages/index.js

```diff
@@ -20,7 +20,7 @@ const applyErrorPageHandlers = pipe(withLoading, withPageWrapper, withContexts);
 export const ArticlePage = pipe(
   applyBasicPageHandlers,
   withVariant,
-)(loadable(() => import(/* webpackPrefetch: true */ './ArticlePage')));
+)(loadable(() => import('./ArticlePage')));
```

Comment:
removed pre-fetching as it's a distraction initially, can be re-added very easily